docs/whitespace.md

```diff
@@ -0,0 +1,96 @@
+@page can-stache.Whitespace Whitespace Control
```

Comment:
I'm not sure this should be in "pages".  I feel like this is closer to control flow and should be in the tags section:

<img width="209" alt="cursor_and_canjs_-_can-stache" src="https://cloud.githubusercontent.com/assets/78602/26026765/5633a976-37c7-11e7-96ba-d969d1845cda.png">


and documented similar to how those are.